src/ptychography/reconstruction/ssb/udf.py

```diff
@@ -1,11 +1,66 @@
 import numpy as np
+import numba
 
 from libertem.udf import UDF
 from libertem.common.container import MaskContainer
 
 from ptychography.reconstruction.ssb.trotters import generate_masks
 
 
+@numba.njit(fastmath=True, cache=True, parallel=True)
```

Comment:
One issue: it would be nice to also set `nogil=True`, so this function can be executed in parallel with other Python threads, should be useful in case of live processing.

Reply:
Done! :-)